src/Build.UnitTests/BuildEnvironmentHelper_Tests.cs

```diff
@@ -59,6 +59,41 @@ public void FindBuildEnvironmentByEnvironmentVariable()
             }
         }
 
+#if RUNTIME_TYPE_NETCORE
+        [Theory(Skip = "NA on .net core")]
+#else
+        [Theory]
+#endif
+        [InlineData(true)]
+        [InlineData(false)]
+        [Trait("Category", "nonlinuxtests")]
+        [Trait("Category", "nonosxtests")]
```

Comment:
```c#
        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp, "No Visual Studio install for netcore")]
        [PlatformSpecific(TestPlatforms.Windows)]
```

As a simplification?

Reply:
Yes. I didn't remember what they were so I copied them from other tests. Replaced throughout the file. Also took out the other tests which were expecting the VS mode under .net core. They were only passing due to how the environment was setup, and would have failed in real life anyway.